targets/targets.mk

```diff
@@ -10,12 +10,20 @@ $(eval $(call GluonTarget,x86,geode))
 $(eval $(call GluonTarget,x86,64))
 
 ifneq ($(BROKEN),)
+GLUON_TARGET_11S_ONLY=1
 $(eval $(call GluonTarget,ar71xx,mikrotik)) # BROKEN: no sysupgrade support
 $(eval $(call GluonTarget,brcm2708,bcm2710)) # BROKEN: Untested
 $(eval $(call GluonTarget,ipq806x)) # BROKEN: Untested
 $(eval $(call GluonTarget,mvebu)) # BROKEN: No AP+IBSS or 11s support
+$(eval $(call GluonTarget,sunxi)) # BROKEN: Untested
+endif
+
+ifeq ($(GLUON_WLAN_MESH),11s)
+GLUON_TARGET_11S_ONLY=1
+endif
+
+ifneq ($(GLUON_TARGET_11S_ONLY),)
 $(eval $(call GluonTarget,ramips,mt7620)) # BROKEN: No AP+IBSS support
 $(eval $(call GluonTarget,ramips,mt7628)) # BROKEN: No AP+IBSS support
 $(eval $(call GluonTarget,ramips,rt305x)) # BROKEN: No AP+IBSS support
-$(eval $(call GluonTarget,sunxi)) # BROKEN: Untested
 endif
```

Comment:
I'm not a fan of these code changes. I would do something like the following (code untested):
```
GLUON_WLAN_MESH_11s := $(filter 11s,$(GLUON_WLAN_MESH))
```
on the top, and
```
ifneq ($(GLUON_WLAN_MESH_11s)$(BROKEN),)
```
at the actual use site.

* The "BROKEN: No AP+IBSS support" comments must be removed.
* targets.mk should be reorderd so that the BROKEN devices are on the bottom again.
* We also need to make similar changes in the ramips-mt7621 target definition, where we currently only build the ER-X image without BROKEN
* Many devices of the ramips targets need a "BROKEN: untested", which was omitted so far, as the whole target was BROKEN. Maybe we should mark all devices of these targes as BROKEN for now and then remove the BROKEN check after we get test feedback.

Reply:
@NeoRaider 
ramips-mt7621: why would we need a change here? i though so at first, too - but ramips-mt7621 isn't broken anymore, please read the comments in there again - or am i wrong?

BROKEN: untested: i already use one or two of those, i can say for ramips-mt7628 vocore2 that it does work nicely.

Reply:
d-link-dir-860l-b1 and zbt-wg3526 have separate BROKEN guards. These should be moved under a 11s||BROKEN guard eventually as well if they work okay.

Reply:
their BROKEN guard is because of the flakey 2,4GHz wifi. this won't be fixed by this PR, but perhaps by updating mt76, see #1296